### ISSUE TITLE
Editor won't launch on Nouvelobs site w/ 'preventDefault' error.

### ISSUE THREAD

**Issue by dwhly** [hypothesis/client#137]
### Steps to reproduce
1. Browse to: https://via.hypothes.is/bibliobs.nouvelobs.com/paroles-d-haiti/20100121.BIB4760/tout-bouge-autour-de-moi.html
2. Select text and annotate.

### Expected behaviour
Editor should appear.

### Actual behaviour
Clicking annotate does not launch the editor.

### Browser/system information
Version:
0.44.0
User agent:
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.116 Safari/537.36
URL:
http://bibliobs.nouvelobs.com/paroles-d-haiti/20100121.BIB4760/tout-bouge-autour-de-moi.html
Username:
dwhly
Date:
26 Sep 2016 06:53:05 -0700

### Additional details
It appears that there is another js app running that is responsive to selection events and there may be a conflict.

Console shows this error: "nobs;manifest.js?_t=1474889101:2 Uncaught TypeError: Cannot read property 'preventDefault' of undefined"

handleCommand	@	hypothes.is/assets/client/scripts/injector.bundle.js?bb3e58:422
r	@	nobs;manifest.js?_t=1474889101:2
(anonymous function)	@	nobs;manifest.js?_t=1474889101:2

This zendesk ticket: https://hypothesis.zendesk.com/inbox/conversations/639

@jeremydean

**Comment by jeremydean** [comment]
Just a quick additional note/clue, selecting text and then instead clicking the annotate button in the sidebar does work here: the composition window opens in the sidebar and the selected text appears as the target.

**Comment by sean-roberts** [comment]
There is definitely an override happening on their site. One that seems to only pass through 1 argument to the click event listener instead of the two that we are binding. It would require a little more digging but this seems like something we can easily rethink how we do the event listeners.

**Comment by sean-roberts** [comment]
So my branch fixes this issue of the preventDefault. But, it uncovered the next breaking problem with that website. The set the `display:none` property on the button that is being clicked in that panel. That will hide the "Annotate" or "Highlight" button in the adder (depending on which one was used). This is them making a big assumption about what is being used there. So additional work will be needed to make sure the display is not changed as well

**Comment by sean-roberts** [comment]
Add assurance of display being set to block in my branch which fixes the second issue they have.

**Comment by robertknight** [comment]
The interfering `Function.prototype.bind` implementation is being installed by MooTools 1.2.5 apparently (evaluate `MooTools.version` in the console), and specifically this code: https://github.com/mootools/mootools-core/blob/3e7459cb4756475a80921bfde5c19a7c7edb64df/Source/Native/Function.js#L28

It looks like this issue applies to any page which uses either MooTools <= 1.2.x OR includes the 1.2.x compatibility code.

Minimal example:

```html
<html>
  <head>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mootools/1.6.0/mootools-core-compat.js"></script>
  </head>
<body>
  <button id="btn">Click me</button>
<script>
function annotate(command, event) {
  console.log(`Annotate should have been called with 2 arguments but was called with ${arguments.length} args`, arguments);
}

var btn = document.getElementById('btn');
btn.addEventListener('click', annotate.bind(null, 'annotate'));
</script>

</body>
</html>
```

**Comment by sean-roberts** [comment]
_Sorry about the commit spamming above_

This is fixed with client release 0.46.0
